tests/common/file_utils_test.py

```diff
@@ -88,7 +102,7 @@ def test_get_from_cache(self):
         set_up_glove(url, self.glove_bytes, change_etag_every=2)
 
         filename = get_from_cache(url, cache_dir=self.TEST_DIR)
-        assert filename == os.path.join(self.TEST_DIR, url_to_filename(url, etag="0"))
+        assert os.path.exists(filename)
```

Comment:
is there a reason for weakening these tests?

Reply:
Ah, no, not anymore. This is a holdover from the previous commit, where `url_to_filename` no longer had any relation to the cached filename. I'll fix it.